app/models/provider_emails_for_application_choices.rb

```diff
@@ -0,0 +1,46 @@
+class ProviderEmailsForApplicationChoices
+  def initialize(choice_ids)
+    @choice_ids = choice_ids
+  end
+
+  def as_hash
+    @_as_hash ||= @choice_ids.reduce({}, &method(:choice_reducer))
+  end
+
+  def as_csv
+    require 'csv'
```

Comment:
Feels weird to `require` inside of a method call but I guess it's fine 🤔

Reply:
i did it because I didn't want to require it unless this was running, as the file will be autoloaded